semantic/src/site/globals/site.overrides

```diff
@@ -546,6 +546,14 @@ a:hover {
 .review-section-row {
   .section-single-row-box-container();
 
+  .ui.grid > .row {
+    justify-content: stretch;
+  }
+
+  // .ui.grid > .column.review-level {
+  //   width: 10%;
+  // }
```

Comment:
Remove commented

Reply:
Cool. I was just trying to make the first column smaller. But let's leave it for a future layout improvement.